library/patient.inc

```diff
@@ -968,8 +968,8 @@ function newPatientData(
     $regdate = ""
 ) {
 
-    $DOB = fixDate($DOB);
-    $regdate = fixDate($regdate);
+    $DOB = DateToYYYYMMDD($DOB);
+    $regdate = DateToYYYYMMDD($regdate);
```

Comment:
I removed above (no fixDate or DateToYYYYMMDD) and are instead doing this from the origin scripts.